src/react/tests/useQueries.test.tsx

```diff
@@ -36,14 +51,145 @@ describe('useQueries', () => {
 
     renderWithClient(queryClient, <Page />)
 
-    await sleep(30)
+    await sleep(10)
 
     expect(results.length).toBe(3)
     expect(results[0]).toMatchObject([{ data: undefined }, { data: undefined }])
     expect(results[1]).toMatchObject([{ data: 1 }, { data: undefined }])
     expect(results[2]).toMatchObject([{ data: 1 }, { data: 2 }])
   })
 
+  it('should return the correct states with suspense', async () => {
```

Comment:
can you maybe add a separate test for only `useErrorBoundary` without `suspense` ?